### DISCUSSION TITLE
system could not be solved: RHS is not finite

### DISCUSSION THREAD

**Discussion by csungeun** [Open-Systems-Pharmacology/Forum#1007] [Questions & Problems] [answered]
There's an error saying, "system could not be solved: RHS is not finite
![error](https://user-images.githubusercontent.com/75017721/140272519-4033bf9b-996a-4464-9c43-538b7d234097.JPG)
"

**Comment by StephanSchaller** [comment]
@csungeun , this might happen if you have a very "stiff" ODE system (i.e., very small concentrations of a molecule (e.g. a binding partner) interacting with very large concentrations of another molecule (e.g. the compound you are modeling)).
Try adjusting your solver settings (reduce tolerances)

**Comment by Yuri05** [comment, marked as answer]
@csungeun Can you reduce the simulation time to e.g. 1 second and repeat the simulation? Do you still get the error?
<img width="550" src="https://user-images.githubusercontent.com/25061876/140729946-faa38d39-6ff8-472d-8de6-3346ff62d009.png">

If you have Matlab, you could also try [this](https://github.com/Open-Systems-Pharmacology/Forum/issues/210#issuecomment-412834236)
